src/haven/Translator.java

```diff
@@ -0,0 +1,107 @@
+package haven;
+
+import org.json.JSONObject;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.io.*;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Translator {
+    public enum Language {
+        ENGLISH("en"),
+        RUSSIAN("ru");
+
+        private final String text;
+
+        private Language(final String text) {
+            this.text = text;
+        }
+
+        @Override
+        public String toString() {
+            return text;
+        }
+    }
+
+    private static String apikey = "trnsl.1.1.20151011T100257Z.56e6087f23957f5d.309d4ea2aae34ee07f33dde22949c8c3cff5c34a";
```

Comment:
Why do you think so? It certainly has some restrictions (1kk characters per month) but I don't think that Yandex will ban our API key because of requests from several IP addresses